content/workshops/ansible_tower/exercise1.4.adoc

```diff
@@ -142,7 +165,9 @@ image::at_project_detail.png[Cred_Detail, 900,caption="Figure 6: ",title="Defini
 
 
 
-*Step 4:* Select SAVE     image:at_save.png[Save,35,25] +
+=== Step 4:
+
+Select SAVE     image:at_save.png[Save,35,25] +
```

Comment:
You don't need the trailing `+` here. The extra line below this forces the new line.